requirements.txt

```diff
@@ -7,7 +7,6 @@ Flask-OAuthlib
 iso8601
 PyJWT
 coveralls
-cryptography
```

Comment:
It does not work locally ...
I will test it now on travis
![image](https://cloud.githubusercontent.com/assets/1217999/15307039/c3249bcc-1bcf-11e6-9277-67c72778bf95.png)